### PR TITLE
Fix README.md to be modern

### PR DESCRIPTION
**Pull request by shibayu36**
README.md had been old, so I fix README.md correctly.

I wrote README.md in reference to https://github.com/mackerelio/go-mackerel-plugin-helper

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,38 +38,45 @@ var graphdef = map[string]mackerelplugin.Graphs{
...
 ```
 
-## Tempfile
-
-`MackerelPlugin` interface has `Tempfile` field. The Tempfile is used to calculate differences in metrics with `Diff: true`.
-If this field is omitted, the filename of the temporaty file is automatically generated from plugin filename.
+## Method
 
-### Default value of Tempfile
+A plugin must implement this interface and the `main` method.
```
Comment (syou6162) [inline README.md]: [nit]interfaces?

File: README.md [hunk 2]
```diff
@@ -24,38 +38,45 @@ var graphdef = map[string]mackerelplugin.Graphs{
...
 ```go
-  helper.Tempfile = *optTempfile
-  if optTempfile == nil {
-    helper.SetTempfileByBasename("YOUR_DEFAULT_FILENAME")
-  }
+func main() {
+	optHost := flag.String("host", "localhost", "Hostname")
+	optPort := flag.String("port", "11211", "Port")
+	optTempfile := flag.String("tempfile", "", "Temp file name")
+    optMetricKeyPrefix := flag.String("metric-key-prefix", "memcached", "Metric Key Prefix")
```
Comment (syou6162) [inline README.md]: [nit]indent seems to be broken

File: README.md [hunk 3]
```diff
@@ -24,38 +38,45 @@ var graphdef = map[string]mackerelplugin.Graphs{
...
+	flag.Parse()
+
+	var memcached MemcachedPlugin
+
+	memcached.Target = fmt.Sprintf("%s:%s", *optHost, *optPort)
+    memcached.prefix = *optMetricKeyPrefix
```
Comment (syou6162) [inline README.md]: [nit]indent seems to be broken

File: README.md (hunk 1 above)
Comment (syou6162) [reply README.md]: Sorry, I misunderstand :bow: It's OK.

**Comment by shibayu36** [comment]
I fixed indent.  I'll merge it after testing.

**Pull request merged**